Intial Microsoft.ApiManagement Swagger Spec

solankisamir opened the pull request:
- Published the 2015-09-15 which is the current Active Version in ARM and Ibiza
- Publishing the 2016-07-07 which we are going to switch in a month.

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

azurecla commented:
Hi __@solankisamir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sasolank). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

solankisamir committed:
Intial Microsoft.ApiManagement Swagger Spec

- Published the 2015-09-15 which is the current Active Version in ARM
and Ibiza
- Publishing the 2016-07-07 which we are going to switch in a month.

solankisamir commented:
Closing the pull request.. do you lot of merge commits.
